src/dcos_e2e/cluster.py

```diff
@@ -90,6 +92,61 @@ def from_nodes(
             cluster_backend=backend,
         )
 
+    @retry(
+        exceptions=(
+            subprocess.CalledProcessError,
+            ValueError,
+            requests.exceptions.ConnectionError,
+        ),
+        tries=500,
+        delay=5,
+    )
+    def _wait_for_adminrouter(
+        self,
+        log_output_live: bool = False,
+    ) -> None:
+        """
+        Wait until all DC/OS systemd units are healthy.
+
+        Args:
+            log_output_live: If `True`, log output of the diagnostics check
+                live. If `True`, stderr is merged into stdout in the return
+                value.
+
+        Raises:
+            ValueError: If Admin Router is not capable yet of serving the CA
+                certificate for the DC/OS cluster.
+
+        """
+        diagnostics_args = [
+            '/opt/mesosphere/bin/dcos-diagnostics',
+            '--diag',
+            '||',
+            '/opt/mesosphere/bin/3dt',
```

Comment:
Let's get rid of the 3dt check - no one is using such an old version of DC/OS with E2E.